src/simulation/m_global_parameters.fpp

```diff
@@ -634,9 +635,6 @@ contains
         ! Gamma/Pi_inf Model ===============================================
         if (model_eqns == 1) then
 
-            ! Setting number of fluids
-            num_fluids = 1
```

Comment:
Can this safely be removed?

Reply:
Yes. There is a check in `m_checker` here https://github.com/MFlowCode/MFC/blob/bbc7211490ba2f1051cfb50cd281fa7a578e49dd/src/simulation/m_checker.fpp#L100-L102